src/components/ContributionInfo.vue

```diff
@@ -67,5 +73,24 @@ export default defineComponent({
 
 .markdown-card-text {
   padding: 16px;
+  text-align: start;
```

Comment:
This makes the CC license image left aligned. Not a big deal but just as an fyi.